### DISCUSSION TITLE
Is Directus able to make a WebService Rest API for OracleDB?

### DISCUSSION THREAD

**Discussion by ldsoulza** [directus/directus#3259] [Archive]
Hello everybody.

I'm studying ways to make a connection from Flutter Apps to Oracle Databases set with Windows Server.

With Directus I'll be able to make this bridge? I need to make the common SQL Query commands (select, update, insert etc.).

I found some tutorials in making the Rest API with NodeJS, and since Directus is NodeJS related maybe would fit for my purposes.

**Comment by rijkvanzanten** [comment]
Heya! We're still wrapping up [full Oracle DB support in Directus](https://github.com/directus/directus/issues?q=is%3Aissue+is%3Aopen+label%3A%22DB%3A+Oracle%22) but afaict Directus should be able to help you out here.

**Reply by rijkvanzanten** [reply]
2022 update: Yes. OracleDB is fully supported
